Assignment 2 completed

reachsneha02 opened the pull request:
<!-- PULL REQUEST TITLE: UofT-DSI | <Module Name> - Assignment <assignment number>-->

## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)

## What did you learn from the changes you have made?

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?

## How were these changes tested?

## A reference to a related issue in your repository (if applicable)

## Checklist
- [ ] I can confirm that my changes are working as intended

Commit:
Completed all parts of assignment 2

Commit:
Slight modifications in the assignment

anjali-deshpande-hub reviewed (changes requested):
Looks good. Just one change is required. The first question asks for using readline or readlines method to load csv into a variable an iterate over the variable to print the lines of a file. Could you modify your code to meet the requirement. Also, the semicolon at the end of print line is not required (line no 21)

Commit:
Updated assignment-2 as per suggestions given;
Used readlines instead of csv.reader, reoved the semicolon after print statement

anjali-deshpande-hub reviewed (approved):
Well done! Thanks!
